### DISCUSSION TITLE
Is there an update mechanism to update the client itself?

### DISCUSSION THREAD

**Discussion by Vysp3r** [Droid-ify/client#699] [General]
I can't seem to find a way to update the client to the latest version.
I also tried downloading the latest version and updating it manually, but I got 'App not installed as package conflicts with an existing package.'

**Comment by kaoneko** [comment]
Did you figure it out in the mean time? You just update it from within Droid-ify like any other app. Droid-ify should be in the list of installed apps.

**Reply by Vysp3r** [reply]
I switched to Obtainium, but yeah I figured it out before switching.
